src/libmongoc/doc/mongoc_database_command.rst

```diff
@@ -32,14 +37,37 @@ Parameters
 ----------
 
 * ``database``: A :symbol:`mongoc_database_t`.
-* ``flags``: A :symbol:`mongoc_query_flags_t`.
-* ``skip``: The number of documents to skip on the server.
-* ``limit``: The maximum number of documents to return from the cursor.
-* ``batch_size``: Attempt to batch results from the server in groups of ``batch_size`` documents.
+* ``flags``: Unused.
+* ``skip``: Unused.
+* ``limit``: Unused.
+* ``batch_size``: Unused.
 * ``command``: A :symbol:`bson:bson_t` containing the command.
-* ``fields``: An optional :symbol:`bson:bson_t` containing the fields to return. ``NULL`` for all fields.
+* ``fields``: Unused.
 * ``read_prefs``: An optional :symbol:`mongoc_read_prefs_t`. Otherwise, the command uses mode ``MONGOC_READ_PRIMARY``.
 
+Migrating
+---------
+
+:symbol:`mongoc_database_command` is deprecated.
+
+The following example uses :symbol:`mongoc_database_command`:
+
+.. literalinclude:: ../examples/migrating.c
+   :language: c
+   :dedent: 6
+   :start-after: // mongoc_database_command ... before ... begin
+   :end-before:  // mongoc_database_command ... before ... end
+   :caption: Before
+
+To migrate, use a non-deprecated alternative, like :symbol:`mongoc_database_command_simple`:
```

Comment:
```suggestion
The above code block may be rewritten to use :symbol:`mongoc_database_command_simple` instead, as shown below:
```
Wording tweaks: indicate the code block below is equivalent to the code block above.

Suggestion also applies to the other doc pages.